fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/directory/AgilityContextTest.java

```diff
@@ -414,6 +418,39 @@ void testAgilityContextAtomicAttribute(boolean useAgile) {
         }
     }
 
+    @Test
+    void testAgilityContextRecoveryPath() {
+        AtomicInteger refInt = new AtomicInteger(0);
+        try (FDBRecordContext context = openContext()) {
+            final AgilityContext agilityContext = getAgilityContext(context, true);
+            final Subspace subspace = path.toSubspace(context);
+            byte[] key = subspace.pack(Tuple.from(2023, 3));
+            byte[] val = Tuple.from(800, "Green eggs and ham", 0).pack();
+            agilityContext.set(key, val);
```

Comment:
We should assert at the end of the test that this was not actually written.

It looks as though there isn't much other testing of the abort/close functionality, so perhaps it would be better to add other tests, rather than adding more assertions to this test.
Additional tests could make it clearer what the expectations are.